.github/workflows/ci.yml

```diff
@@ -345,26 +345,24 @@ jobs:
 
         cargo run ${{ env.HOSTARGS }} --features assembly --target=$HOST_TARGET test_assembly ${{ env.ARGS }}
 
-    - name: Run Cargo Dinghy
+    - name: Launch XCode Simulator
```

Comment:
I've thought about using something like [futureware-tech/simulator-action](https://github.com/marketplace/actions/launch-ios-simulator) for this type of thing. I'm on the fence about it because the few lines of bash is easier to run locally when things break.